packages/engine/src/audio/systems/AudioSystem.ts

```diff
@@ -121,5 +121,6 @@ export default async function AudioSystem(world: World) {
     for (const entity of playQuery.enter(world)) {
       whenReady(() => playSoundEffect(entity))
     }
+    removeActionReceptor(AudioSettingReceptor)
```

Comment:
this is not right, we dont need to worry about removing the receptor for now. this return function is a system not an unmount hook